omnibus/config/software/python2.rb

```diff
@@ -18,7 +18,7 @@
 name "python2"
 
 if ohai["platform"] != "windows"
-  default_version "2.7.16"
+  default_version "2.7.17"
```

Comment:
there's something wrong with the diff on github, this change is already on master. Not a blocker to merge the PR, I'm just surprised github didn't update the diff correctly when the base of the PR was changed to `master`...